npm/pkg/dataplane/types.go

```diff
@@ -8,6 +8,7 @@ import (
 type GenericDataplane interface {
 	InitializeDataPlane() error
 	ResetDataPlane() error
+	GetIPSet(setName string) *ipsets.IPSet
```

Comment:
As discussed offline, I feel this shouldn't be part of the generic dataplane interface. Perhaps the gsp should use the dataplane struct instead of the GenericDataplane interface

Reply:
This is a function we will need for DPmocks for UTs and also for GSProcessor. So kept it as a higher level interface method. In future, we can this function for other utilities, like debugging or observability.

Reply:
GetIPSet is being used by GSProcessors, DP, to figure out what is existing set goal state and to compute diff. We are also using this in UTs. 

Atm, we do not use this function in normal coupled mode.